src/blocks/renderers/customForm.tsx

```diff
@@ -173,6 +173,7 @@ async function setData(
         data: {
           id: recordId,
           data: cleanValues,
+          // Using shallow strategy to support partial data views
```

Comment:
What does "partial data views" mean in this context? Doesn't merge strategy have more to do with how the data is changed, not how it's viewed? Just worried that this comment is adding more confusion than clarity from an outside perspective.

Reply:
Does this read more clearly?
```suggestion
          // Using shallow strategy to support partial data forms
          // In case when a form contains (and submits) only a subset of all the fields of a record,
          // the fields missing in the form will not be removed from the DB record
```

Reply:
It is about the view in this case. The whole record can contain data `{ foo: 1, bar: 2}`, a form renders a single field `foo` and on submitting it pushes to the server `{ foo: 42 }`. We want to preserve the value of `bar` in the database.